Dockerfile.agnos

```diff
@@ -78,24 +78,28 @@ RUN /tmp/agnos/openpilot_python_dependencies.sh
 COPY --from=agnos-compiler-ffmpeg /tmp/ffmpeg.deb /tmp/ffmpeg.deb
 RUN cd /tmp && apt-get -o Dpkg::Options::="--force-overwrite" install -yq ./ffmpeg.deb
 
-RUN export PATH="/usr/local/pyenv/bin:/usr/local/pyenv/shims:$PATH" && \
-    export PYENV_ROOT="/usr/local/pyenv" && \
-    eval "$(pyenv init -)" && \
-    eval "$(pyenv virtualenv-init -)" && \
+RUN export PATH="$HOME/.cargo/bin:$PATH" && \
+    export UV_BIN="$HOME/.cargo/bin" && \
     export MAKEFLAGS="-j$(nproc)" && \
-    pip install pyqt5-sip==12.12.1 && \
-    pip install pyqt5==5.15.9 --verbose --config-settings --confirm-license= && \
-    pyenv rehash
+    source .venv/bin/activate && \
+    .venv/bin/python --version && \
+    uv pip install pyqt6-sip==13.8.0 && \
+    uv pip install pyqt6==6.7.1 --verbose --config-settings="--confirm-license="
```

Comment:
```suggestion
    uv pip install pyqt5==5.15.9 --verbose --config-settings="--confirm-license=" --no-deps
```

Reply:
pyqt5 can be installed from sources. I guess we should stay with pyqt5 for now.

Reply:
I have reverted to pyqt5.